### DISCUSSION TITLE
Need Help Installing "headless-documents" Package

### DISCUSSION THREAD

**Discussion by petarvlajic** [pimcore/pimcore#16395] [Q&A / Support] [answered]
Hello Pimcore Community,

I hope you're well. I'm having trouble installing the "headless-documents" package using the provided documentation. Despite following the steps, I encounter an error 
**"Could not find a matching version of package pimcore/headless-documents. Check the package spelling, your version  
   constraint and that the package is available in a stability which matches your minimum-stability (stable).  "**

1. tried with "composer require pimcore/headless-documents:*" to find and version that suits
2. Checked https://packagist.org/ for pimcore/headless-documents but that package doesn't exist...

Pimcore: 11.0.4 v
Data-hub: 1.6 v

headless documents documentation link: https://pimcore.com/docs/platform/Headless_Documents/Headless_Documents

 Your assistance in resolving this issue would be greatly appreciated. Any known workarounds or additional information not covered in the documentation would be helpful.

Thank you,
Petar

**Comment by dpfaffenbauer** [comment, marked as answer]
@petarvlajic headless Documents are part of the enterprise subscription.
